### PR TITLE
add follow user in profile

### PR DESCRIPTION
**Pull request by tungblvt**
==> Title: add follow user in profile

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/14769

### CODE REVIEW

**Comment by tungblvt** [comment]
ready

**Comment by tungblvt** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class UsersController < ApplicationController
   before_action :authenticate_user!
-  before_action :find_user
+  before_action :find_user, only: %i(show)
 
   def show
     @activities = Result.where(user_id: @user.id).includes(:course).page(params[:page])
```
Comment (huuphuc93) [inline app/controllers/users_controller.rb]: Viết scope cho where(user_id: @user.id) rồi gọi ra ở đây

**Comment by tungblvt** [comment]
ready

**Pull request merged**